From 97907ba386344101abb14d9eb3baf96cba8edfcc Mon Sep 17 00:00:00 2001
From: LuK1337 <priv.luk@gmail.com>
Date: Wed, 20 Sep 2017 12:31:07 +0200
Subject: [PATCH 13/14] sepolicy: Add /data/vendor/time label for old oreo
 blobs

 * Old O blobs moved to /data/vendor/time but there will be issue
   that system time will be reset after ota upgrade, so Qualcomm
   moved back to /data/time in newer release. Label /data/vendor/time
   for compatibility with those old O blobs.

Change-Id: Ifecf4f0d9218eb47c7478a6a2e3edb6648c4ed89
---
 legacy-common/file_contexts | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/legacy-common/file_contexts b/legacy-common/file_contexts
index 5433780..6fd17e0 100644
--- a/legacy-common/file_contexts
+++ b/legacy-common/file_contexts
@@ -17,3 +17,6 @@
 
 # Subsys restart
 /sys/devices(/platform)?/soc\.0/[a-z0-9\.:]+,[a-z0-9\-]+/subsys[0-9]+/name    u:object_r:sysfs_ssr:s0
+
+# Time services
+/data/vendor/time(/.*)?               u:object_r:time_data_file:s0
-- 
2.7.4

